### PR TITLE
Minor wording updates

### PR DESCRIPTION
**Pull request by cchiffersMSFT**
minor wording updates within the VPN Network recommendations to call out that customers should use split tunnelling to route Teams media Traffic.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/microsoft-teams-online-call-flows.md [hunk 1]
```diff
@@ -82,12 +82,12 @@ Teams media flows connectivity is implemented using standard IETF Interactive Co
 
 ### Technologies that are not recommended with Microsoft Teams
 
-**VPN network**. It is not recommended for media traffic (or flow 2'). The VPN client should use split VPN and route media traffic like any external non-VPN user, as specified in [Enabling Lync media to bypass a VPN tunnel](https://techcommunity.microsoft.com/t5/Skype-for-Business-Blog/Enabling-Lync-Media-to-Bypass-a-VPN-Tunnel/ba-p/620210).
+**VPN network**. It is not recommended for media traffic (or flow 2'). The VPN client should use split tunnelling and route Teams media traffic like any external non-VPN user, as specified in [Enabling Lync media to bypass a VPN tunnel](https://techcommunity.microsoft.com/t5/Skype-for-Business-Blog/Enabling-Lync-Media-to-Bypass-a-VPN-Tunnel/ba-p/620210).
```
Comment (JohanFreelancer9) [inline Teams/microsoft-teams-online-call-flows.md:85]:
```suggestion
**VPN network**. It is not recommended for media traffic (or flow 2'). The VPN client should use split tunneling and route Teams media traffic like any external non-VPN user, as specified in [Enabling Lync media to bypass a VPN tunnel](https://techcommunity.microsoft.com/t5/Skype-for-Business-Blog/Enabling-Lync-Media-to-Bypass-a-VPN-Tunnel/ba-p/620210).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cchiffersMSFT please commit the minor typo correction. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**